index.js

```diff
@@ -0,0 +1,33 @@
+const options = [
+    { emoji: '😃', count: 0 },
+    { emoji: '😄', count: 0 },
+    { emoji: '😊', count: 0 },
+    { emoji: '😐', count: 0 },
+    { emoji: '😕', count: 0 }
+  ];
+  
+  function vote(index) {
+    options[index].count++;
+    updateResults();
+  }
+  
+  function updateResults() {
+    const resultsContainer = document.getElementById('results');
+    resultsContainer.innerHTML = '';
+  
+    for (let i = 0; i < options.length; i++) {
+      const emojiElement = document.createElement('div');
+      emojiElement.classList.add('emoji');
+      emojiElement.textContent = options[i].emoji;
+  
+      const voteCount = document.createElement('span');
+      voteCount.classList.add('vote-count');
+      voteCount.textContent = options[i].count;
+      emojiElement.appendChild(voteCount);
+  
+      emojiElement.addEventListener('click', () => vote(i));
```

Comment:
You add listener for each smile, there are should be one for parent element

Look at this solution, and fix

```
<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>Voting smiles</title>
    <style>
        .parent {
            display:flex;
            margin: 30px 0 0;
            justify-content: center;
            user-select: none;
        }

        .emoji {
            padding: 10px;
            font-size: 60px;
        }

        .emoji__picture {
            cursor: pointer;
        }

        .emoji__count {
            text-align: center;
        }
    </style>
</head>
<body>
    <div class="parent">
        <div class="emoji">
            <div class="emoji__picture">&#129315;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128578;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128528;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128577;</div>
            <div class="emoji__count">0</div>
        </div>
        <div class="emoji">
            <div class="emoji__picture">&#128557;</div>
            <div class="emoji__count">0</div>
        </div>
        <!-- UNCOMMENT THIS CODE -->
        <!--<div class="emoji">
            <div class="emoji__picture">&#129488;</div>
            <div class="emoji__count">0</div>
        </div>-->
    </div>
    <script>
      const parent = document.querySelector(".parent");

      parent.addEventListener("click", (event) => {
        if (event.target.closest(".emoji__picture")) {
          const nextElementSibling = event.target.nextElementSibling;

          nextElementSibling.textContent = Number(nextElementSibling.textContent) + 1;
        }
      });
    </script>
</body>
</html>
```